File: packages/ios/Sources/CivUI/CivTabNavItem.swift
// CivUI — CivTabNavItem for SwiftUI
// Single item in a tab-nav. Mirrors civ-tab-nav-item.
//
// Placeholder body — the prop surface satisfies schema parity. The
// implementation should wire the platform-native equivalent. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivTabNavItem: View {
    /// label
    public var label: String

    /// href
    public var href: String

    /// current
    public var current: Bool

    /// disabled
    public var disabled: Bool

    /// Called for analytics tracking (parallels civ-analytics).
    public var onAnalytics: ((String, [String: Any]?) -> Void)?

    public init(
        label: String = "",
        href: String = "",
        current: Bool = false,
        disabled: Bool = false,
        onAnalytics: ((String, [String: Any]?) -> Void)? = nil
    ) {
        self.label = label
        self.href = href
        self.current = current
        self.disabled = disabled
        self.onAnalytics = onAnalytics
    }

    public var body: some View {
        EmptyView()
    }
}

File: packages/android/src/main/kotlin/gov/civui/components/CivTabNavItem.kt
// CivUI — CivTabNavItem for Jetpack Compose
// Single item in a tab-nav. Mirrors civ-tab-nav-item.
//
// Placeholder body — the prop surface satisfies schema parity. The
// implementation should wire the Compose-native equivalent. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivTabNavItem(
    label: String = "",
    href: String = "",
    current: Boolean = false,
    disabled: Boolean = false,
    modifier: Modifier = Modifier,
    onAnalytics: ((event: String, data: Map<String, Any>?) -> Unit)? = null,
    content: @Composable () -> Unit = {}
) {
    Box(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
        content()
    }
}
